ckan/views/dataset.py

```diff
@@ -944,7 +944,7 @@ def follow(package_type: str, id: str) -> Response:
     return h.redirect_to(u'{}.read'.format(package_type), id=id)
 
 
-def unfollow(package_type: str, id: str) -> Response:
+def unfollow(package_type: str, id: str) -> Union[Response, str]:
```

Comment:
In which cases we are returning `str`?

Reply:
I m not sure, i think it was a fix for `Check types / typecheck (pull_request) ` tests, can you help with this one?